nvflare/private/fed/utils/fed_utils.py

```diff
@@ -383,11 +391,11 @@ def get_return_code(process, job_id, workspace, logger):
         except Exception:
             logger.warning(
                 f"Could not get the return_code from {rc_file} of the job:{job_id}, "
-                f"Return the RC from the process:{process.pid}"
+                f"Return the RC from the job_handle:{job_handle}"
             )
-            return_code = process.poll()
+            return_code = job_handle.poll()
```

Comment:
This return_code should be standard, not handle-specific now. Should change the poll() method in all launchers to return standard code.

Reply:
changed to standard return code.